source/configuring-ember/preparing-for-upgrade.md

```diff
@@ -0,0 +1,154 @@
+A valueable attribute of the EmberJS framework is its use of [Semantic Versioning](http://semver.org/), that allows projects to keep up with
```

Comment:
maybe better words would be to say "..use of Semantic Versioning to aid projects in keeping up..."